README.md

```diff
@@ -258,6 +258,10 @@ The following sample retrieves Google+ profile of the authenticated user.
 var google = require('googleapis');
 var plus = google.plus('v1');
 var OAuth2 = google.auth.OAuth2;
+
+// WARNING: it's important to keep filled  YOUR_CLIENT_ID,
+// YOUR_CLIENT_SECRET, YOUR_REDIRECT_URL for  for use automatically refreshed
```

Comment:
Double _for_. Can you delete one of them?

Reply:
sorry, deleted